Source/Core/UICommon/UICommon.cpp

```diff
@@ -108,6 +110,12 @@ static void InitCustomPaths()
 #endif
 }
 
+static void RefreshConfig()
+{
+  Common::SetEnableAlert(Config::Get(Config::MAIN_USE_PANIC_HANDLERS));
```

Comment:
I think this doesn't work like you intended it. If you have Use Panic Handlers on, then click 'ignore for this session' and then change *any* setting, it would re-enable the panic handlers. We probably need to split the current session boolean from the setting boolean here.

Reply:
That's what the second commit fixes.

Reply:
Ooh I see, you do this via the config layering system, I missed that. Makes sense.